examples/single_chip/camera_web_server/main/app_httpd.c

```diff
@@ -59,6 +69,7 @@ static const char* _STREAM_PART = "Content-Type: image/jpeg\r\nContent-Length: %
 
 httpd_handle_t stream_httpd = NULL;
 httpd_handle_t camera_httpd = NULL;
+bool isStreaming = false;
```

Comment:
Please include this variable in the macro

Reply:
Makes sense. Fixed with 4b563c2